packages/commcare/src/Adaptor.js

```diff
@@ -29,6 +29,52 @@ export function execute(...operations) {
   };
 }
 
+/**
+ * Make a get request to any commcare endpoint
+ * @public
+ * @example
+ * get(
+ *    "case"
+ *    {
+ *      limit: 1,
+ *      offset:0,
+ *    }
+ * )
+ * // The returned objects will be written into state.data,
+ * // and the meta object will be written to state.response
+ * // along with the status code and returned headers.
+ * @function
+ * @param {string} path - Path to resource
+ * @param {Object} params - Optional request params such as limit and offset.
+ * @returns {Operation}
+ */
+export function get(path, params = {}) {
```

Comment:
Ah, I made a mistake here -  I forgot about the callback!

The third parameter should be an optional `callback` function, just like in `openmrs`. The callback should be passed into `prepareNextState`  (which might need updating to support it)